scripts/prepare-unity.sh

```diff
@@ -41,13 +41,19 @@ else
   echo "Could not find a Unity $UNITY_PINNED_VERSION installation."
 fi
 
-if [ -d "$IMPROBABLE_UNITY_VERSIONED/PlaybackEngines/LinuxStandaloneSupport" ] || \
-   [ -d "$IMPROBABLE_UNITY_VERSIONED/PlaybackEngines/AndroidPlayer" ] || \
-   [ -d "$IMPROBABLE_UNITY_VERSIONED/PlaybackEngines/iOSSupport" ] || \
-   [ -d "$IMPROBABLE_UNITY_VERSIONED/PlaybackEngines/WindowsStandaloneSupport" ] || \
-   [ -d "$IMPROBABLE_UNITY_VERSIONED/PlaybackEngines/MacStandaloneSupport" ]
+if isWindows; then
+  PLAYBACK_ENGINE_ROOT="$IMPROBABLE_UNITY_VERSIONED/Editor/Data/PlaybackEngines"
+elif isMacOS; then
+  PLAYBACK_ENGINE_ROOT="$IMPROBABLE_UNITY_VERSIONED/PlaybackEngines"
+fi
+
+if [ -d "$PLAYBACK_ENGINE_ROOT/LinuxStandaloneSupport" ] && \
+   [ -d "$PLAYBACK_ENGINE_ROOT/AndroidPlayer" ] && \
+   [ -d "$PLAYBACK_ENGINE_ROOT/iOSSupport" ] && \
+   [ -d "$PLAYBACK_ENGINE_ROOT/WindowsStandaloneSupport" ] && \
+   [ -d "$PLAYBACK_ENGINE_ROOT/MacStandaloneSupport" ]
```

Comment:
`    [ -d "$PLAYBACK_ENGINE_ROOT/MacStandaloneSupport" ] ` could still be invalid for Mac Editor, AFAIK it should be installed within the .app - unless our installer may do a redundant build support on top?

Reply:
yup - on mac the `MacStandaloneSupport` directory is inside the .app at `/Applications/Unity/Unity/Contents/PlaybackEngines/MacStandaloneSupport`

in which case I should probably not check for this directory when setting up it's a mac agent